dspace-api/src/test/data/dspaceFolder/config/spring/api/solr-services.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+           http://www.springframework.org/schema/context
+           http://www.springframework.org/schema/context/spring-context-2.5.xsd"
+       default-autowire-candidates="*Service,*DAO,javax.sql.DataSource">
+
+    <context:annotation-config /> <!-- allows us to use spring annotations in beans -->
+
+    <bean class="org.dspace.discovery.MockSolrServiceImpl" id="org.dspace.discovery.SearchService"/>
+
+    <alias name="org.dspace.discovery.SearchService" alias="org.dspace.discovery.IndexingService"/>
+
+    <!--<bean class="org.dspace.discovery.SolrServiceIndexOutputPlugin" id="solrServiceIndexOutputPlugin"/>-->
+
+    <!-- Statistics services are both lazy loaded (by name), as you are likely just using ONE of them and not both -->
+    <bean id="elasticSearchLoggerService" class="org.dspace.statistics.ElasticSearchLoggerServiceImpl" lazy-init="true"/>
```

Comment:
is this really required? the plan is to withdrawn ES support in DSpace7

Reply:
I moved these services from [here](https://github.com/DSpace/DSpace/blob/dspace-6.2/dspace/config/spring/api/core-services.xml#L95) to this new file so that we can override them in the integration tests.

I suggest we remove the `elasticSearchLoggerService` bean once we remove the complete Elastic Search implementation. I don't want to partly cleanup Elastic Search now with the risk of forgetting stuff.

Reply:
ok, I agree